Error deploying cloud platform to GKE cluster

Hi, I am trying to follow this tutorial on your website to setup Confluent Platform on my GKE cluster . 
https://docs.confluent.io/current/tutorials/examples/kubernetes/gke-base/docs/index.html

However, when I get to the section labelled **Run** and execute the command 

`make demo`

I get the following error

```
✔ found curl
✔ found tar
✔ found kubectl
✔ found helm
✔ found jq
✔ found gcloud
✔ gke-base dependencies verified
Extracting operator to /Users/al/Desktop/cluster/examples/kubernetes/common/cp/operator/20200310-v0.142.1/
tar: Error opening archive: Unrecognized archive format
make[1]: *** [get-helm-bundle] Error 1
make: *** [demo] Error 2
```

Not sure why I am getting this error, if it makes any difference I will note that I skipped the sections prior in the tutorial as I am doing this to a pre existing cluster and not making one from scratch.

Cheers

--- a/kubernetes/common/Makefile
+++ b/kubernetes/common/Makefile
@@ -44,10 +44,10 @@ get-helm-bundle: #_ Download the Confluent Operator Helm bundle locally in prepa
 ifeq (,$(wildcard $(OPERATOR_DOWNLOAD_PATH)))
 	@mkdir -p $(dir $(OPERATOR_DOWNLOAD_PATH))
 	@echo "downloading Confluent Operator to $(OPERATOR_DOWNLOAD_PATH)"
-	@curl -s https://platform-ops-bin.s3-us-west-1.amazonaws.com/operator/confluent-operator-$(OPERATOR_HELM_CHART_VERSION).tar.gz --output $(OPERATOR_DOWNLOAD_PATH)
+	@curl -s https://platform-ops-bin.s3-us-west-1.amazonaws.com/operator/confluent-operator-$(OPERATOR_BUNDLE_VERSION).tar.gz --output $(OPERATOR_DOWNLOAD_PATH)
 endif
 ifeq (,$(wildcard $(OPERATOR_PATH)))
-	@mkdir -p $(dir $(OPERATOR_DOWNLOAD_PATH))$(OPERATOR_HELM_CHART_VERSION)/
+	@mkdir -p $(dir $(OPERATOR_DOWNLOAD_PATH))$(OPERATOR_BUNDLE_VERSION)/
 	@echo "Extracting operator to $(OPERATOR_PATH)"
 	@tar xf $(OPERATOR_DOWNLOAD_PATH) -C $(OPERATOR_PATH)
 endif

